package.json

```diff
@@ -15,5 +15,8 @@
   "bugs": {
     "url": "https://github.com/YvesGanza/Hello-world/issues"
   },
-  "homepage": "https://github.com/YvesGanza/Hello-world#readme"
+  "homepage": "https://github.com/YvesGanza/Hello-world#readme",
+  "devDependencies": {
+    "eslint": "^8.6.0"
+  }
 }
```

Comment:
[OPTIONAL] This file is not needed for this project. It will be nice if you could remove it.